feat(changelog): object-storage-added-new-bucket-policy-version-allowing 2023-11-08 (#2384)
scaleway/docs-content: changelog/november2023/2023-11-08-object-storage-added-new-bucket-policy-version-allowing.mdx

---
title: New bucket policy version allowing to set up granular access to ressources is now in GA!
status: added
author:
    fullname: 'Join the #object-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-11-08
category: storage
product: object-storage
---

> With the version 2023-04-17 of bucket policies, you can grant users and applications granular permissions on specific buckets and objects.

Reviewer: ```suggestion
With version 2023-04-17 of bucket policies, you can grant users and applications granular permissions to specific buckets and objects.
```

> Check out our documentation to discover this new bucket policy version https://www.scaleway.com/en/docs/storage/object/api-cli/bucket-policy/, and learn how to combine IAM and bucket policies to set up granular access to Object Storage! https://www.scaleway.com/en/docs/storage/object/api-cli/combining-iam-and-object-storage/.

Reviewer 2: ```suggestion
Check out our documentation to discover this new [bucket policy version](https://www.scaleway.com/en/docs/storage/object/api-cli/bucket-policy/) , and learn [how to combine IAM and bucket policies](https://www.scaleway.com/en/docs/storage/object/api-cli/combining-iam-and-object-storage/) to set up granular access to Object Storage!
```
